iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/Networks.java

```diff
@@ -450,4 +450,25 @@ public static AbstractReportNodeContext allowReportNodeContextMultiThreadAccess(
         }
         return newReportNodeContext != null ? newReportNodeContext : reportNodeContext;
     }
+
+    /**
+     * This method replaces the "input" status of switches, tap changer position and shunt section count by their solved values if they are present.
+     */
+    public static void applySolvedValues(Network network) {
```

Comment:
Should we also add a method that copies only the new solved values to make migration easier?